packages/documentation/cypress/snapshots/utilities/opacity.snapshot.ts

```diff
@@ -2,6 +2,6 @@ describe('Opacity', () => {
   it('opacity', () => {
     cy.visit('/iframe.html?id=snapshots--opacity');
     cy.get('.opacity-example', { timeout: 30000 }).should('be.visible');
```

Comment:
`.opacity-example` no longer exist in the snapshot page: https://preview-3754--swisspost-design-system-next.netlify.app/iframe.html?id=snapshots--opacity&viewMode=story